umbral/utils.py

```diff
@@ -28,14 +28,14 @@ def poly_eval(coeff, x):
     return result
 
 
-def kdf(ecpoint, key_length):
+def kdf(ecpoint, key_length, salt=None, info=None):
```

Comment:
I'm thinking this should be called `derive_key_from_point` since the input is presumed to be a point. Hmmm, perhaps this belongs to `umbral.Point`?

Reply:
Hmmm, yeah that seems reasonable.